lib/elixir/pages/Guards.md

```diff
@@ -40,8 +41,8 @@ In the example above, we show how guards can be used in function clauses. There
   * function clauses:
 
     ```elixir
-    def foo(term) when is_integer(term), do: term
-    def foo(term) when is_float(term), do: round(term)
+    def format(term) when is_integer(term), do: term
+    def format(term) when is_float(term), do: round(term)
```

Comment:
I don't think `format` makes any more sense than `foo`.

Reply:
maybe `format_number` ?

Reply:
Let's rewrite it to:

```elixir
def type(term) when is_integer(term), do: :integer
def type(term) when is_float(term), do: :float
```

?

Reply:
done